meteor/client/ui/RundownList/RundownListItemView.tsx

```diff
@@ -111,9 +114,15 @@ export default withTranslation()(function RundownListItemView(props: Translated<
 			<span className="rundown-list-item__text">
 				<JonasFormattedTime timestamp={rundown.modified} t={t} />
 			</span>
-			<span className="rundown-list-item__text">
-				<RundownShelfLayoutSelection rundown={props.rundown} />
-			</span>
+			{rundownLayouts.some((l) => l.exposeAsShelf || l.exposeAsStandalone) && (
```

Comment:
Ooh, I've never heard of `.some()`! Thanks!

![giphy](https://user-images.githubusercontent.com/3635991/109285345-4150cb80-7821-11eb-9913-3e10c746a243.gif)